### PR TITLE
added skills to employee

### PR DESCRIPTION
**Pull request by priyanhsu10**

### CODE REVIEW

File: src/com/company/practice/Employee.java [hunk 1]
```diff
@@ -42,4 +42,11 @@ public  String getIdName(){
         System.out.println("Name and Id : "+getName()+ getId() );
         return getName()+ getId();
     }
+    public  void addSkills(String[] skills){
+
+        for (String skill : skills) {
+            System.out.println(skill);
```
Comment (WebSamurai) [inline src/com/company/practice/Employee.java]: print skill : skill name

Comment (WebSamurai) [reply src/com/company/practice/Employee.java]: print skill : skill name

**Review by WebSamurai — CHANGES_REQUESTED** [review]

**Comment by priyanhsu10** [comment]
fix review comments

File: src/com/company/practice/Employee.java [hunk 2]
```diff
@@ -42,4 +42,11 @@ public  String getIdName(){
         System.out.println("Name and Id : "+getName()+ getId() );
         return getName()+ getId();
     }
+    public  void addSkills(String[] skills){
+
+        for (String skill : skills) {
+            System.out.println( "Skill:" + skill);
```
Comment (WebSamurai) [inline src/com/company/practice/Employee.java:48]: add one more print statements

**Review by WebSamurai — CHANGES_REQUESTED** [review]

**Comment by priyanhsu10** [comment]
added new comment line

**Pull request merged**